### PR TITLE
Added the HorizontalIconAlignment property to the button.

### PR DESCRIPTION
**Pull request by m0lDaViA**
Added the HorizontalIconAlignment property to the button.
You can now choose, if you want to show the icon left or right to the content.

## Pull request type

Please check the type of change your PR introduces:

- [x] Update
- [x] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-

### CODE REVIEW

File: src/Wpf.Ui/Controls/Button/Button.cs [hunk 1]
```diff
@@ -13,16 +13,16 @@ namespace Wpf.Ui.Controls;
 /// </summary>
 /// <example>
 /// <code lang="xml">
-/// &lt;ui:Button
```
Comment (pomianowski) [inline src/Wpf.Ui/Controls/Button/Button.cs:16]: Examples should not use special characters
https://wpfui.lepo.co/api/Wpf.Ui.Controls.Button.html

File: src/Wpf.Ui/Controls/Button/Button.cs [hunk 2]
```diff
@@ -181,6 +188,21 @@ public Brush PressedBorderBrush
...
+    /// <summary>
+    /// Gets or sets whether the icon should be shown left or right from the content.
+    /// </summary>
+    public HorizontalIconAlignments HorizontalIconAlignment
+    {
+        get => (HorizontalIconAlignments)GetValue(HorizontalIconAlignmentProperty);
+        set => SetValue(HorizontalIconAlignmentProperty, value);
+    }
+
+    public enum HorizontalIconAlignments
```
Comment (pomianowski) [inline src/Wpf.Ui/Controls/Button/Button.cs:203]: Missing docs for public API.

Comment (pomianowski) [inline src/Wpf.Ui/Controls/Button/Button.cs:203]: I would extract it to a separate file

**Review by pomianowski — CHANGES_REQUESTED** [review]

File: src/Wpf.Ui/Controls/Button/Button.cs (hunk 2 above)
Comment (m0lDaViA) [reply src/Wpf.Ui/Controls/Button/Button.cs:203]: Alright.

File: src/Wpf.Ui/Controls/Button/Button.cs (hunk 1 above)
Comment (m0lDaViA) [reply src/Wpf.Ui/Controls/Button/Button.cs:16]: Because? I mean even vs is using it totally fine.

Comment (m0lDaViA) [reply src/Wpf.Ui/Controls/Button/Button.cs:16]: Oh and btw & is a special character :)

File: src/Wpf.Ui/Controls/Button/Button.cs (hunk 2 above)
Comment (m0lDaViA) [reply src/Wpf.Ui/Controls/Button/Button.cs:203]: Any suggestion for the docs?

**Comment by Jay-o-Way** [comment]
Hm, the weird "alignment" usually should like it is placed inside of a space. Like: text on a page is aligned inside/between the borders of the page. Not _next to_ the page.
Here, is more like a position _relative_ to something else.

**Comment by m0lDaViA** [comment]
> Hm, the word "alignment" usually should like it is placed inside of a space. Like: text on a page is aligned inside/between the borders of the page. Not _next to_ the page. Here, is more like a position _relative_ to something else.

But it isn't relative, that's the difference.

**Comment by Jay-o-Way** [comment]
What I mean is, the icon is placed to the left/right of the content/text. Therefore I think "HorizontalIconPlacement" would be a good (or even more accurate) wording.

By the way, I assume this works well in RTL language?

**Comment by m0lDaViA** [comment]
> What I mean is, the icon is placed to the left/right of the content/text. Therefore I think "HorizontalIconPlacement" would be a good (or even more accurate) wording.
> 
> By the way, I assume this works well in RTL language?

In this case I followed the standard alignment that MS uses.

**Comment by m0lDaViA** [comment]
#1120 & #887 can check the last commit.

**Comment by m0lDaViA** [comment]
I don't know why, but my vs is bugged rn. I cannot create visible styles and my formatter does instead of formatting only opened files all .xaml files.
Sorry for that.

**Pull request closed without merging**